### ISSUE TITLE
AttributeError in InlineMultiQuestion

### ISSUE THREAD

**Issue by inducer** [inducer/relate#118]
@dzhuang, can you take a look? (Encountered in @lukeolson's class.)

```
Internal Server Error: /course/cs357-f15/flow-session/59150/finish/
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./relate/utils.py", line 165, in wrapper
    return retry_transaction(f, args, kwargs, max_tries=max_tries)
  File "./relate/utils.py", line 154, in retry_transaction
    return f(*args, **kwargs)
  File "./course/utils.py", line 484, in wrapper
    response = f(pctx, *args, **kwargs)
  File "./course/flow.py", line 1602, in finish_flow_session_view
    now_datetime=now_datetime)
  File "/usr/lib/python2.7/dist-packages/django/utils/decorators.py", line 145, in inner
    return func(*args, **kwargs)
  File "./course/flow.py", line 517, in finish_flow_session
    force_regrade=force_regrade)
  File "/usr/lib/python2.7/dist-packages/django/utils/decorators.py", line 145, in inner
    return func(*args, **kwargs)
  File "./course/flow.py", line 492, in grade_page_visits
    graded_at_git_commit_sha=fctx.course_commit_sha)
  File "./course/flow.py", line 133, in grade_page_visit
    visit.answer, grade_data=grade_data)
  File "./course/page/inline.py", line 843, in grade
    total_weight += self.answer_instance_list[idx].weight
AttributeError: 'InlineMultiQuestion' object has no attribute 'answer_instance_list'
```

**Comment by inducer** [comment]
Should be fixed in 0ca0a53d3cedad59baf796bdbc9f565c816c89f6.

**Comment by dzhuang** [comment]
Sorry for late response, and thank you for the fix!
